src/components/message/message-card/MessageCard.tsx

```diff
@@ -47,11 +47,173 @@ export interface RecoveredProps {
 	setViewType: (viewType: MessageViewType) => void;
 }
 
+export interface MinimalMessageCardProps extends OwnProps, RecoveredProps {
+	hoverMessage?: () => void;
+	unhoverMessage?: () => void;
+	isAttached?: boolean;
+	isBookmarked?: boolean;
+	isHighlighted?: boolean;
+	isSoftFiltered?: boolean;
+	isContentBeautified?: boolean;
+	toogleMessagePin?: () => void;
+	isEmbedded?: boolean;
+}
+
 interface Props extends OwnProps, RecoveredProps {}
 
-function MessageCardBase({ message, viewType, setViewType }: Props) {
+export function MinimalMessageCard({
```

Comment:
let's refactor MessageCardBase into arrow function and wrap it in observer. 
then delete `const MessageCard = observer(MessageCardBase);`
then rename MessageCardBase to MessageCard and MinimalMessageCard to MessageCardBase.